File: support-utils/tests/src/com/hendraanggrian/support/utils/test/ThemesTest.java
package com.hendraanggrian.support.utils.test;

import android.support.test.runner.AndroidJUnit4;

import com.hendraanggrian.support.utils.content.Themes;

import org.junit.Assert;
import org.junit.Test;
import org.junit.runner.RunWith;

import static android.support.test.InstrumentationRegistry.getTargetContext;
import static org.junit.Assert.assertNotEquals;

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
@RunWith(AndroidJUnit4.class)
public class ThemesTest {

    @Test
    public void test() {
        Assert.assertNotEquals(Themes.getColorOrDefault(getTargetContext(), android.R.attr.colorAccent, 0), 0);
    }
}

File: tests/src/com/hendraanggrian/support/utils/test/ThemesTest.kt
package com.hendraanggrian.support.utils.test

import android.support.test.InstrumentationRegistry.getTargetContext
import android.support.test.runner.AndroidJUnit4
import com.hendraanggrian.support.utils.content.colorAttr
import org.junit.Assert.assertNotEquals
import org.junit.Test
import org.junit.runner.RunWith

/**
 * @author Hendra Anggrian (hendraanggrian@gmail.com)
 */
@RunWith(AndroidJUnit4::class)
class ThemesTest {

    @Test
    fun test() {
        assertNotEquals(R.attr.colorAccent.colorAttr(getTargetContext()), 0)
    }
}